include/hlslib/common/OpenCL.h

```diff
@@ -499,6 +502,97 @@ class Buffer {
   Buffer(Context &context, size_t nElements)
       : Buffer(context, MemoryBank::unspecified, nElements) {}
 
+#ifdef HLSLIB_XILINX
+  /// Allocate DDR or HBM but don't perform any transfers.
+  Buffer(Context &context, StorageType storageType, uint8_t bankIndex, size_t nElements)
```

Comment:
Why a `uint8_t` and not just an `int`?

Reply:
Changed to int and added checks for >= 0